site/src/components/Popover/Popover.tsx

```diff
@@ -1,10 +1,12 @@
 import MuiPopover, {
 	type PopoverProps as MuiPopoverProps,
-	// biome-ignore lint/nursery/noRestrictedImports: Used as base component
+	// biome-ignore lint/nursery/noRestrictedImports: This is the base component that our custom popover is based on
```

Comment:
I might have Biome configured wrong because my Biome plugin is flagging all of these comments as not doing anything

Accidentally removed this at first, but that's definitely a me problem, right?

Reply:
I think it doesn't like that our biome.json config is in the site/ directory 🙄 I hate that extensions never bother to handle subdirectories well.